scripts/loadMovies.js

```diff
@@ -0,0 +1,35 @@
+import fs from "fs";
+import { PrismaClient } from "@prisma/client";
+
+const prisma = new PrismaClient();
+
+function loadMovies() {
+  const data = fs.readFileSync("../ml-1m/movies.dat", "utf-8");
+  const movies = data.split("\n");
+  let promises = movies.map((movie) => {
```

Comment:
I don't think let is needed here since there is no reassignment. 
```
const promisses = 
```

In general, always try to avoid let and mutations.